README.md

```diff
@@ -454,6 +454,37 @@ AS confederation parameters. Valid values are String, keyword 'default'.
 ##### `enforce_first_as`
 Enable/Disable enforces the neighbor autonomous system to be the first AS number listed in the AS path attribute for eBGP. Valid values are 'true', 'false', and 'default'.
 
+##### `event_history_cli`
+Enable/Disable cli event history buffer. Valid values are 'true', 'false', and 'default'.
+
+##### `event_history_cli_size`
+Set cli event history buffer size. Valid values are 'small', 'medium', 'large', 'disable' or 'default'.
```

Comment:
During the design phase Andi and I actually discussed doing this as a single property. I don't recall a compelling reason to do it one way or the other at the time, unless it had to do with lack of a size option on some platforms, though I'm not sure XR even supports event-history. If there's no good reason for it then I'm fine with removing the boolean prop.